storage_visualization/disk_usage.py

```diff
@@ -54,6 +54,8 @@ def aggregate_level(name: str) -> str:
         return name[: ht_index + 3]
     if (mt_index := name.find('.mt/')) != -1:
         return name[: mt_index + 3]
+    if (mt_index := name.find('.vds/')) != -1:
+        return name[: mt_index + 4]
```

Comment:
```suggestion
    if (vds_index := name.find('.vds/')) != -1:
        return name[: vds_index + 4]
```

Reply:
spotted this as soon as I published the PR 🤦